images/Dockerfile.ubuntu-20-lts

```diff
@@ -40,14 +42,15 @@ RUN apt install -y \
     squashfs-tools \
     sudo \
     systemd \
-    && apt-get clean
+    systemd-timesyncd \
+    xz-utils \
+  && apt-get clean \
```

Comment:
the clean is not needed anymore: https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105